src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -316,33 +316,18 @@ CATCH_RETURN()
 
 #pragma endregion
 
-#pragma region DxRenderer
-
-HRESULT AtlasEngine::Enable() noexcept
-{
-    return S_OK;
-}
+#pragma region getter
 
 [[nodiscard]] std::wstring_view AtlasEngine::GetPixelShaderPath() noexcept
 {
     return _api.s->misc->customPixelShaderPath;
 }
 
-[[nodiscard]] std::wstring_view AtlasEngine::GetPixelShaderImagePath() noexcept

```

Comment:
Did this just entirely disappear?

Reply:
oh I see, it did. But no one was using it. gotcha.

Reply:
Yup there's another one, related to DPI, which was used but only in a single place so I just inlined it there.